brainframe_qt/ui/resources/settings.py

```diff
@@ -1,130 +1,75 @@
 import sys
-import typing
-from enum import Enum
-from typing import Any
-
-import pendulum
-from PyQt5.QtCore import QSettings
-from pendulum.tz.timezone import Timezone
-from pubsus import PubSubMixin
-
-
-class Setting(PubSubMixin):
```

Comment:
Just curious- is there any other usage of PubSubMixin in the client?

Reply:
Guess not. I'll remove the dependency